src/App.jsx

```diff
@@ -25,22 +26,25 @@ const App = () => {
     }, [isLoading])
 
     return (
-        <>
-            <nav>
+        <div>
+            <nav className="bg-red-100 w-screen">
```

Comment:
Depending on your scroll styling you might get horizontal scrolling as w-screen is 100vw, which does not care about the scrollbar gutter. Do you need width styling at all? nav is a block element, so it will take as much space as possible